Changelog.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   especially turn maximisation problems into minimisation ones using a scaling of `-1`.
 * Introduce a `ManifoldConstrainedSetObjective`
 * Introduce a `projected_gradient_method`
+* Allow setting `AbstractManifoldObjective` through JuMP
```

Comment:
I would have loved to edit this but one this that is really horrible on web interface editing is these multiline suggestions, I never can get them to work as I hoped for.

We already released 0.5.12 a few days ago since I needed that for another algorithm so this has to be moved to a new 0.5.13 section (“unreleased” we addd the date later in the tutorial)